packages/client/vite.config.js

```diff
@@ -37,6 +37,9 @@ export default defineConfig(({ mode }) => {
     css: {
       devSourcemap: true
     },
-    mode: mode
+    mode: mode,
+    server: {
```

Comment:
does this turn off hot reload

Reply:
It does. I got errors in the browser console and the changes were not applied while having `hmr` (Hot Module Reload) turned on while on the `/login` route. Could be only on my machine though, so please let me know if it does work on anyone else's machine.. then I can turn it back on and I'll need to investigate more on what the problem is.